rubicon_ml/client/project.py

```diff
@@ -379,11 +393,23 @@ def archive(self, experiments: Optional[List[Experiment]] = None, remote_rubicon
             if not isinstance(remote_rubicon, Rubicon):
                 raise ValueError("`remote_rubicon` must be of type `rubicon_ml.client.Rubicon`")
             else:
-                return self.repository._archive(
-                    self.name, experiments, remote_rubicon.repository.root_dir
-                )
+                res = []
```

Comment:
lets remove these changes to the archiving and leave it as-is for now. if a user archives a multi-repo to one s3 bucket, we don't need to archive it more than once - it'd just keep overwriting itself with the same data